packages/graphql/tests/schema/directives/relationship-aggregate.test.ts

```diff
@@ -0,0 +1,650 @@
+        expect(movieActorsAggregate).toBeDefined();
+
+        const queryFields = schema.getQueryType()?.getFields() as GraphQLFieldMap<any, any>;
+
+        const movies = queryFields["movies"];
+        const actors = queryFields["actors"];
+
+        expect(movies).toBeDefined();
+        expect(actors).toBeDefined();
+
+        const moviesConnection = queryFields["moviesConnection"];
+        const actorsConnection = queryFields["actorsConnection"];
+
+        expect(moviesConnection).toBeDefined();
+        expect(actorsConnection).toBeDefined();
+
+        const moviesAggregate = queryFields["moviesAggregate"];
+        const actorsAggregate = queryFields["actorsAggregate"];
+
+        expect(moviesAggregate).toBeDefined();
+        expect(actorsAggregate).toBeUndefined();
+
+        const movieActorActorsAggregationSelection = schema.getType(
+            "MovieActorActorsAggregationSelection"
+        ) as GraphQLObjectType;
+        expect(movieActorActorsAggregationSelection).toBeDefined();
+    });
+
+    test("should work in conjunction with @query aggregate:true and @relationship aggregate:false", async () => {
+        const typeDefs = gql`
+            type Actor @query(aggregate: true) {
+                username: String!
+                password: String!
+            }
+
+            type Movie {
+                title: String
+                actors: [Actor!]! @relationship(type: "ACTED_IN", direction: IN, aggregate: false)
+            }
+        `;
+
+        const neoSchema = new Neo4jGraphQL({ typeDefs });
+        const schema = await neoSchema.getSchema();
+        const movieType = schema.getType("Movie") as GraphQLObjectType;
+        expect(movieType).toBeDefined();
+
+        const movieFields = movieType.getFields();
+        const movieActorsAggregate = movieFields["actorsAggregate"];
+        expect(movieActorsAggregate).toBeUndefined();
+
+        const queryFields = schema.getQueryType()?.getFields() as GraphQLFieldMap<any, any>;
+
+        const movies = queryFields["movies"];
+        const actors = queryFields["actors"];
+
+        expect(movies).toBeDefined();
+        expect(actors).toBeDefined();
+
+        const moviesConnection = queryFields["moviesConnection"];
+        const actorsConnection = queryFields["actorsConnection"];
+
+        expect(moviesConnection).toBeDefined();
+        expect(actorsConnection).toBeDefined();
+
+        const moviesAggregate = queryFields["moviesAggregate"];
+        const actorsAggregate = queryFields["actorsAggregate"];
+
+        expect(moviesAggregate).toBeDefined();
+        expect(actorsAggregate).toBeDefined();
+
+        const movieActorActorsAggregationSelection = schema.getType(
+            "MovieActorActorsAggregationSelection"
+        ) as GraphQLObjectType;
+        expect(movieActorActorsAggregationSelection).toBeUndefined();
+    });
+
+    test("snapshot test with aggregate argument set as false", async () => {
```

Comment:
Can you add a snapshot for true as well?

Reply:
Sure!